packages/cmf/src/actions/http.js

```diff
@@ -1,4 +1,71 @@
+import has from 'lodash/has';
+
 import { HTTP_METHODS } from '../middlewares/http';
+import {
+	ACTION_TYPE_HTTP_REQUEST,
+	ACTION_TYPE_HTTP_RESPONSE,
+	ACTION_TYPE_HTTP_ERRORS,
+	ACTION_TYPE_HTTP_REDUCER_ERROR,
+} from '../middlewares/http/constants';
+
+export const DEFAULT_HTTP_HEADERS = {
+	Accept: 'application/json',
+	'Content-Type': 'application/json',
+};
+
+export function isHTTPRequest(action) {
+	return action.type in HTTP_METHODS || has(action, 'cmf.http');
+}
+
+export function httpError(error) {
+	return {
+		type: ACTION_TYPE_HTTP_ERRORS,
+		error,
+	};
+}
+
+export function httpRequest(url, config) {
+	return {
+		type: ACTION_TYPE_HTTP_REQUEST,
+		url,
+		config,
+	};
+}
+
+export function httpReducerError(error, action) {
```

Comment:
i think the naming including `reducer` is wrong, since its an action

Reply:
This is an action triggered by the middleware when an error JS happens during the fetch. It is correct.